.swiftlint.yml

```diff
@@ -42,9 +42,9 @@ only_rules:
   - trailing_semicolon
 
   # Lines should not have trailing whitespace.
-  - trailing_whitespace
+  # - trailing_whitespace
 
-  - vertical_whitespace
+  # - vertical_whitespace
```

Comment:
Disabling this, so that we stop chasing whitespaces or extra newlines, all over the place

Reply:
🙇